docs/custom-install.md

```diff
@@ -106,165 +105,83 @@ According to that, the ISO for the example above is expected to include the `/oe
-name: "Install extra drivers"
-stages:
-  before-install:
-    # Preload data to the persistent storage
-    # During installation persistent partition is mounted at /run/cos/persistent
-    - commands:
-        - rsync -a /run/initramfs/live/data/ /run/cos/persistent
-  after-install-chroot:
-    # extra_drivers folder is at `/usr/local/extra_drivers` from the OS image chroot
-    - commands:
-      - rpm -iv /usr/local/extra_drivers/some_driver.rpm
+apiVersion: elemental.cattle.io/v1beta1
+kind: SeedImage
+metadata:
+  name: custom-seed
+  namespace: fleet-default
+spec:
+  ...
+  cloud-config:
+    stages:
+      post-install:
+      - name: "Run custom script after installation"
+        commands:
+        - |
+          echo "This is a custom script"
+          echo "For instance, this could be used to handle extra drives for an LVM group"
+      boot:
+      - name: "Add proxy setup for the installation media"
+        files:
+        - path: /etc/sysconfig/proxy
+          permissions: 0664
+          content: |
+            PROXY_ENABLED="yes"
+            HTTP_PROXY=http://<MY_PROXY>:<MY_PORT>
+            HTTPS_PROXY=https://<MY_PROXY>:<MY_PORT>
+            NO_PROXY="localhost, 127.0.0.1"
 ```
 
-Note the installation hooks only cover installation procedures, for upgrades equivalent
-`before-upgrade` and/or `after-upgrade-chroot` should be defined.
-
-### Adding extra LVM volume group disks during the installation
-
-This example is covering the setup of an host with multiple disks and some of them used
-as part of an LVM setup.
-
-As an example, we have an host with three disks (`/dev/sda`, `/dev/sdb`
-and `/dev/sdc`). 
-
-The first disk is used for a regular Elemental installation
-and the other remaining two are used as part of a LVM group where arbitrary logical volumes
-are created, formatted and mounted at boot time via an extended `fstab` file.
-
-For this example, the following files are required:
-
-* additional `clout-init` files included in the installed system
-* additional installation hooks to prepare the LVM volumes during the installation
-
-Let's create an `overlay` directory to create the directory tree that needs to be
-added into the ISO root. In that case the `overlay` directory could contain:
+### Custom Elemental client configuration file
 
-```yaml showLineNumbers
-overlay/
-  oem/
-    lvm_volumes_in_fstab.yaml
-  iso-config/
-    lvm_volumes_hook.yaml
-```
+[Elemental client](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental.md) `install`, `upgrade` and `reset` commands can be configured with a [custom configuration file](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/) located by default in `/etc/elemental/config.yaml`.
+If you have multiple yaml files, you need to add them in the `/etc/elemental/config.d` directory.
 
-The installation hook `overlay/iso-config/lvm_volumes_hook.yaml`:
+A simple example to set hooks location could be:
```

Comment:
Not really sure what this sentence mean. Does it miss an example path such as `/etc/elemental/config.d/extra-partition.yaml` for the sample below?

Reply:
🤦🏽 this a leftover from somewhere. It should not be here, I am reviewing it.